packages/serverless/package-lock.json

```diff
@@ -72,6 +72,21 @@
 			"integrity": "sha512-wdppn25U8z/2yiaT6YGquE6X8sSv7hNMWSXYSSU1jGv/yd6XqjXgTDJ8KP4NgjTXfJ3GbRjeeb8RTV7a/VpM+w==",
 			"dev": true
 		},
+		"@instana/core": {
+			"version": "1.132.2",
+			"resolved": "https://registry.npmjs.org/@instana/core/-/core-1.132.2.tgz",
+			"integrity": "sha512-zbvqYeDcpODerZ98jh2WbpvzTH/4neM1XJAsosvKpPfng1cr0Q3gPW1glcaRQ5CwhLJVitv0M0lyYTiNpzcVyg==",
+			"requires": {
+				"async-hook-jl": "^1.7.6",
+				"cls-bluebird": "^2.1.0",
+				"lru-cache": "^6.0.0",
+				"methods": "^1.1.2",
+				"opentracing": "^0.14.5",
+				"redis-commands": "^1.7.0",
+				"semver": "7.3.5",
+				"shimmer": "1.2.1"
+			}
+		},
```

Comment:
*issue:* This declaration is wrong here. package-lock.json files in the individual packages must not declare dependencies to other `@instana` packages as those are managed by lerna. I would suggest to revert this file.

Reply:
Good catch! Was probably touched when I was adding symbolic links in the beginning of my test.
🙈